src/applications/gi/components/search/KeywordSearch.jsx

```diff
@@ -82,22 +62,38 @@ export class KeywordSearch extends React.Component {
           htmlFor="institution-search">
           {this.props.label}
         </label>
-        <Autosuggest
-          getSuggestionValue={this.clickedSuggestionValue}
-          highlightFirstSuggestion
-          onSuggestionsClearRequested={this.props.onClearAutocompleteSuggestions}
-          onSuggestionSelected={this.handleSuggestionSelected}
-          onSuggestionsFetchRequested={this.handleFetchSuggestion}
-          renderSuggestion={this.renderSuggestion}
-          shouldRenderSuggestions={this.shouldRenderSuggestions}
-          suggestions={suggestions}
-          inputProps={{
-            value: searchTerm,
-            onChange: this.handleChange,
-            onKeyUp: this.handleKeyUp,
-            'aria-labelledby': 'institution-search-label',
-            onBlur: this.handleInputBlur,
-          }}/>
+        <Downshift inputValue={searchTerm} onSelect={(item) => this.handleSuggestionSelected(item)}>
```

Comment:
Playing around with this and VoiceOver, I think we need to provide a function for the `itemToString` prop, so that it doesn't read "Object object" in some cases. This is what worked for me:

```js
function itemToString(item) {
  if (item && typeof item === 'object') {
    return item.label;
  }
   
  return item;
}
```

Reply:
kk